Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
Traceback (most recent call last):
  File "/root/discord-bot/hell-bot/v1/main-app/main.py", line 1, in <module>
    import discord  # Импорт библиотеки для работы с Discord API
ModuleNotFoundError: No module named 'discord'
